pandas/core/frame.py

```diff
@@ -7134,7 +7133,6 @@ def _join_compat(
         self, other, on=None, how="left", lsuffix="", rsuffix="", sort=False
     ):
         from pandas.core.reshape.merge import merge
```

Comment:
can the `merge` import also be moved up?

Reply:
Was hoping so but doesn't appear to be the case. I *think* its intertwined with some of the Categorical stuff so might make sense if / when I can resolve that space

Reply:
Great.

Categorical is used in a lot of places where I think/hope a less high-powered tool could be used, and doing so couple simplify the dependency structure of the code base quite a bit.